app/controllers/user/sessions_controller.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module User
+  class SessionsController < Devise::SessionsController
+    before_action :configure_sign_in_params, only: [:create]
+
+    # GET /resource/sign_in
+
+    # POST /resource/sign_in
+    def create
+      user = User.find_by_email(params[:email])
+      if user&.authenticate(params[:password])
+        session[:user_id] = user.id
+        redirect_to root_url
```

Comment:
It should be [this](https://github.com/ita-social-projects/ZeroWaste/blob/master/app/controllers/application_controller.rb#L18)

Reply:
Changed